krun.py

```diff
@@ -248,6 +250,9 @@ def main(parser):
         if rc > 0:
```

Comment:
In UNIX anything non-zero is considered an error.

I ma have screwed this up too. I had to merge some 4 conflicts by hand, so that might be the source of the error.